utils/sampler/uniform_replacer.go

```diff
@@ -30,23 +28,19 @@ func (m defaultMap) get(key uint64, defaultVal uint64) uint64 {
 //
 // Sampling is performed in O(count) time and O(count) space.
 type uniformReplacer struct {
-	rng        rng
-	seededRNG  rng
+	rng        *rng
+	seededRNG  *rng
 	length     uint64
 	drawn      defaultMap
 	drawsCount uint64
 }
 
-func (s *uniformReplacer) Initialize(length uint64) error {
-	if length > math.MaxInt64 {
-		return errOutOfRange
-	}
+func (s *uniformReplacer) Initialize(length uint64) {
 	s.rng = globalRNG
 	s.seededRNG = newRNG()
 	s.length = length
 	s.drawn = make(defaultMap)
 	s.drawsCount = 0
-	return nil
 }
 
 func (s *uniformReplacer) Sample(count int) ([]uint64, error) {
```

Comment:
This will panic if `count = -1`
```
s := uniformReplacer{}
s.Initialize(1)
s.Sample(-1)
```

Reply:
I feel like panicking is pretty reasonable when requesting a negative number of values